judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/AssociationRelationshipsTest.java

```diff
@@ -183,13 +183,7 @@ public void testTraverse() {
             "REQ-ENT-012"
     })
     public void testDeletingRequiredRelationThrowsException() {
-        IllegalStateException thrown = assertThrows(
-                IllegalStateException.class,
-                () -> entityCDao.delete(entityC)
-        );
 
-        assertTrue(thrown.getMessage().contains("There is mandatory references which is not removable"));
-        assertTrue(thrown.getMessage().contains("#singleRequiredConA"));
     }
```

Comment:
Why was this deleted?

Reply:
Fix it

Reply:
[1a37435](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/175/commits/1a37435b20fb4d2843ff2f08ba668cdab23b636b)